ResearchKit/Common/ORKFormItemCell.m

```diff
@@ -416,6 +416,25 @@ - (BOOL)textFieldShouldReturn:(UITextField *)textField {
     return YES;
 }
 
+- (BOOL)textField:(UITextField *)textField shouldChangeCharactersInRange:(NSRange)range replacementString:(NSString *)string {
```

Comment:
`ORKFormItemTextFieldBasedCell` is the base class, seems we don't need to touch here.